### PR TITLE
Organize data, states and requests into corresponding components instead of handling everything in Drawer

### PR DESCRIPTION
**Pull request by erikasv**

### CODE REVIEW

File: src/search/Drawer.jsx [hunk 1]
```diff
@@ -277,63 +72,50 @@ class Drawer extends React.Component {
...
+          <div>
+            <Landscape
+              area={area}
+              geofenceId={area.id === 'pa' ? geofence.name : geofence.id}
+              matchColor={matchColor}
+              handlerSwitchLayer={handlerSwitchLayer}
+              handlerClickOnGraph={handlerClickOnGraph}
+            />
+          </div>
+          <div className="graphcard">
```
Comment (alegnaaived) [inline src/search/Drawer.jsx:105]: It is not showed:

image

Comment (cazapatamar) [reply src/search/Drawer.jsx:105]: That's strange. I solved it in a past PR

Comment (erikasv) [reply src/search/Drawer.jsx:105]: Well it actually doesn't show in develop either :disappointed:

Comment (erikasv) [reply src/search/Drawer.jsx:103]: The issue raised when the accordion pannel was disabled if the area wasn't an EA. Solved in: [e718588](https://github.com/PEM-Humboldt/biotablero-frontend/pull/321/commits/e718588a8091affc50d510705ee38cd5f80ef708)

File: src/landscape/humanFootprint/TimelineFootprint.jsx [hunk 2]
```diff
@@ -155,20 +187,23 @@ class TimelineFootprint extends React.Component {
...
+                onClickGraphHandler(selection);
+              }}
             />
           </h2>
-          {hfTimelineArea && hfTimelineArea.type !== 'Total' && (
+          {selectedEcosystem && (
             <div>
               <h6>
-                {`${hfTimelineArea.type} dentro de la unidad de consulta`}
+                {`${this.getLabel(selectedEcosystem.type)} dentro de la unidad de consulta`}
```
[the remark this answers is not in the document]
Comment (erikasv) [reply src/landscape/humanFootprint/TimelineFootprint.jsx:203]: In that case, the color of the line should reset as well, in develop the line kept its selection, so I created [this issue](https://github.com/PEM-Humboldt/biotablero-frontend/issues/325) to decide one behavior and change it later. Feel free to complement it.

[the remark this answers is not in the document]
Comment (alegnaaived) [reply src/landscape/humanFootprint/TimelineFootprint.jsx:203]: Agree with this new issue

**Pull request merged**